stl/inc/chrono

```diff
@@ -2209,6 +2221,2949 @@ namespace chrono {
 
         return hours{_Ret};
     }
+
+    // [time.zone.info]
+
+    // STRUCT sys_info
+    struct sys_info {
+        sys_seconds begin;
+        sys_seconds end;
+        seconds offset;
+        minutes save;
+        string abbrev;
+    };
+
+    // STRUCT local_info
+    struct local_info {
+        static constexpr int unique      = 0;
+        static constexpr int nonexistent = 1;
+        static constexpr int ambiguous   = 2;
+
+        int result;
+        sys_info first;
+        sys_info second;
+    };
+
+    // CLASS nonexistent_local_time
+    class nonexistent_local_time : public runtime_error {
+    public:
+        template <class _Duration>
+        nonexistent_local_time(const local_time<_Duration>&, const local_info&)
+            : runtime_error("TRANSITION: work in progress") {}
+    };
+
+    // CLASS ambiguous_local_time
+    class ambiguous_local_time : public runtime_error {
+    public:
+        template <class _Duration>
+        ambiguous_local_time(const local_time<_Duration>&, const local_info&)
+            : runtime_error("TRANSITION: work in progress") {}
+    };
+
+    // [time.zone.timezone]
+
+    // ENUM CLASS choose
+    enum class choose { earliest, latest };
+
+    // CLASS time_zone
+    class time_zone {
+    public:
+        explicit time_zone(string_view _Name_) : _Name(_Name_) {}

```

Comment:
+1 for both. Upon looking at this again, the potential for users to write `time_zone tz{"PDT"};` is dangerously high.

Reply:
> Upon looking at this again, the potential for users to write `time_zone tz{"PDT"};` is dangerously high.

Yes, it would be nice to make it easier for users to avoid accidentally (or intentionally) using a non-portable initialization.

Reply:
I noticed that the data member we're constructing is `string _Name;`, so `noexcept` would be unwise here.